### PR TITLE
Misc docs updates

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

**Review by carldebilly — APPROVED** [review]
LGTM

File: doc/articles/features/image.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+uid: Uno.Features.Image
+---
+
+# The Image Control
+
+This page details the specific aspects of the `Image` control for Uno Platform. General information about `Image` is available WinUI [official documentation](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.image).
```
Comment (Youssef1313) [inline doc/articles/features/image.md:7]:
```suggestion
This page details the specific aspects of the `Image` control for Uno Platform. General information about `Image` is available in WinUI [official documentation](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.image).

```

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**